pinecone-mixin/dashboards.libsonnet

```
@@ -0,0 +1,25 @@
+local g = import '../g.libsonnet';
+
+{
+  new(this):
+    {
+      'pinecone-overview.json':
+        g.dashboard.new(this.config.dashboardNamePrefix + 'Pinecone overview')
```

Minor thing, but for consistency let's use the nameprefix 
```suggestion
        g.dashboard.new(this.config.dashboardNamePrefix + ' overview')
```